docs/manpage.rst

```diff
@@ -387,12 +387,13 @@ It does so by leveraging the selected system's environment modules system.
    This option can also be set using the :envvar:`RFM_UNLOAD_MODULES` environment variable or the :js:attr:`unload_modules` general configuration parameter.
 
 
-.. option:: --unuse-module-path=PATH
+.. option:: --module-path=PATH
 
-   Unuse module path ``PATH`` before acting on any tests.
-   This option may be specified multiple times, in which case all specified modules paths will be unused in order.
+   Use module path ``PATH`` before acting on any tests.
```

Comment:
My problem with the `--module-path=/path/to/modules` is that it is a bit confusing. One could think that this would set the whole `MODULEPATH`. That's why I suggest to have the following three options:

1. `+/path/to/modules` -> `module use`
2. `-/path/to/modules` -> `module unuse`
3. `/path/to/modules` -> override completely `MODULEPATH`.

Since, as I suggest in the following, you need not provide a corresponding configuration parameter and/or environment variable, it should be quite straightforward to implement this logic.